deepchem/molnet/load_function/load_dataset_template.py

```diff
@@ -38,15 +38,16 @@
 
 def load_mydataset(
     featurizer: Featurizer = DEFAULT_FEATURIZERS['RawFeaturizer'],
-    transformers: Tuple[Transformer] = (
-        DEFAULT_TRANSFORMERS['PowerTransformer']),
+    transformers: List[Transformer] = [
+        DEFAULT_TRANSFORMERS['PowerTransformer']
+    ],
     splitter: Splitter = DEFAULT_SPLITTERS['RandomSplitter'],
     reload: bool = True,
     data_dir: Optional[str] = None,
     save_dir: Optional[str] = None,
-    featurizer_kwargs: Optional[Dict[str, object]] = None,
-    splitter_kwargs: Optional[Dict[str, object]] = None,
-    transformer_kwargs: Optional[Dict[str, Dict[str, object]]] = None,
+    featurizer_kwargs: Optional[Dict[str, object]] = {},
+    splitter_kwargs: Optional[Dict[str, object]] = {},
+    transformer_kwargs: Optional[Dict[str, Dict[str, object]]] = {},
```

Comment:
Since you've switched the default value to being an empty dict instead of None, do these still need to be marked as Optional?